docs/develop-component-checklist.md

```diff
@@ -0,0 +1,23 @@
+# Developing a component - things to remember
+
+##Don't forget..
+
+- test file, and a JS spec file if there is any JS involved
+- put complex logic into a helper
+- print stylesheet
+
+##A component should..
```

Comment:
This implies all components should have a bottom margin and tracking, which isn't always needed? Perhaps it's ok to implement a bottom margin without it being used anywhere yet, but not tracking.